.github/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 
 ### Documentation
 
+* fixed 2 typos in the state preparation tutorial
+[#321](https://github.com/XanaduAI/pennylane/pull/321)
```

Comment:
Note that in ReST formatting, bullet point continuations need to match the first indentation
```suggestion
   [#321](https://github.com/XanaduAI/pennylane/pull/321)
```